src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceBeforeCheck.java

```diff
@@ -70,6 +83,19 @@
  *   &lt;property name=&quot;allowLineBreaks&quot; value=&quot;true&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * Map .Entry e = null; // violation, whitespace before '.' is not allowed
+ * Map.Entry f = null; // OK
+ * int foo
+ * ; // OK, linebreak is allowed
+ * Customer c = new Customer(); // Customer is a user defined class
+ * c . getCustomer(); // violation, whitespace before '.' is not allowed
+ * c
+ *     .setFirstName("John")
+ *     .setLastName("Doe")
```

Comment:
please make code formatted like:
```
c.setFirstName("John")
 .setLastName("Doe")
```
indentation is 1 spaces, to keep dots aligned (as people usually do with chained calls)

as we do not have that much options in Check, please make two examples blocks have the same code, so user will easily see how behvior changes when allowLineBreaks is activated.

Reply:
> please make two examples blocks have the same code, so user will easily see how behavior changes when allowLineBreaks is activated.

Is it properly executed?